src/fmt.rs

```diff
@@ -1,6 +1,6 @@
 use std::str::FromStr;
 
-use crate::toml_edit::{Document, Item, Table, Value};
+use toml_edit::{Document, Item, Table, Value};
 
 /// The config file for formatting toml after sorting.
 ///
```

Comment:
line 57 above the doc comments for `pub crlf: bool` could you add
```
// NOTE: this is only used in main, fmt doesn't set the line endings
```
(I forget this every time I look through this code base) Thanks.